packages/expo-camera/plugin/src/withCamera.ts

```diff
@@ -3,20 +3,31 @@ import {
   type ConfigPlugin,
   createRunOncePlugin,
   IOSConfig,
+  withPodfileProperties,
 } from 'expo/config-plugins';
 
 const pkg = require('expo-camera/package.json');
 
 const CAMERA_USAGE = 'Allow $(PRODUCT_NAME) to access your camera';
 const MICROPHONE_USAGE = 'Allow $(PRODUCT_NAME) to access your microphone';
+const BARCODE_SCANNER_PODFILE_KEY = 'expo-camera.barcode-scanner-enabled';
```

Comment:
It doesn't matter that much, but I would use a different key. Something like: `expo.camera.barcode-scanner-enabled`